docs/mobile/repo-structure.md

```diff
@@ -0,0 +1,162 @@
+    status: LIVE
+    main_id: 5261
+    lessons:
+      - key: 5b0d45a2-6e81-44e5-b4a5-dc57ab3a50f9
+        title: Beginning your SQL journey
+        description: Learn the basics of SQL.
+        practice: false
+        fileName: chapter1/lesson1.yml
+      - key: 5dbb1040-3ce4-40f6-a97c-f2eabf346309
+        title: Learning to count
+        description: Data science is mostly counting things.
+        practice: false
+        fileName: chapter1/lesson2.yml
+      - key: eca415ab-fbd9-4f39-9930-724af8e10e36
+        title: Practice
+        description: "Practice what you've learned."
+        practice: true
+        fileName: chapter1/practice.yml
+
+assets:
+  - type: dataset
+    key: people
+    url: assets/datasets/people_small.csv
+    tabs:
+      table:
+        title: People
+  - type: dataset
+    key: films
+    url: assets/datasets/films_small.csv
+    tabs:
+      table:
+        title: Films
+  - type: dataset
+    key: reviews
+    url: assets/datasets/reviews.csv
+    tabs:
+      table:
+        title: Reviews
+  - type: dataset
+    key: roles
+    url: assets/datasets/roles.csv
+    tabs:
+      table:
+        title: Roles
+```
+
+The general structure is at least somewhat self-explanatory, but here's a
+breakdown:
+
+| Field              | Sub Field     | Sub Field     | Explanation                                               |
+|--------------------|---------------|---------------|-----------------------------------------------------------|
+| `authoringVersion` |               |               | Should be set to `2` the current authoring format version |
+| `title`            |               |               | Title of the mobile course                                |
+| `description`      |               |               | Description of the course                                 |
+| `key`              |               |               | A unique UUID for the course                              |
+| `badge_url`        |               |               | URL of the course badge image                             |
+| `status`           |               |               | Set to `LIVE` when the course is ready to be launched     |
+| `technology_key`   |               |               | One of `R`, `PYTHON`, or `SQL`                            |
+| `assets`           | `type`        |               | One of `image` or `dataset`                               |
+|                    | `key`         |               | How will this asset be referenced in exercise YAML?       |
+|                    | `url`         |               | The path to the asset (relative to the root directory)    |
+|                    | `tabs`        | `file`        |                                                           |
+|                    |               | `table`       |                                                           |
+| `chapters`         | `key`         |               | A unique UID for the chapter                           |
```

Comment:
This out of alignment pipe bugs me way more than it ought to.

Reply:
Its an appropriate reaction.